docs/reference/testing.md

```
@@ -36,7 +36,36 @@ When `ry test` is run without arguments, it:
+
+Use the `@it` and `@describe` directives to define test cases as ordinary named functions:
+
+```
+@it("test case name")
+function test_add():
+    expect(1 + 2).to_eq(3)
+```
+
+Group related tests using `@describe`:
+
+```
```

markdownlint-cli2 (0.22.0)

[warning] Fenced code blocks should have a language specified

(MD040, fenced-code-language)